server-spi/src/main/java/org/keycloak/models/UserProvider.java

```diff
@@ -57,6 +58,9 @@
 
     UserModel getServiceAccount(ClientModel client);
     List<UserModel> getUsers(RealmModel realm, boolean includeServiceAccounts);
+    default Stream<UserModel> getUsersStream(RealmModel realm, boolean includeServiceAccounts) {
+        return getUsers(realm, includeServiceAccounts).stream();
+    }
```

Comment:
@sguilhen Sorry for confusion, I will remove this. 

To explain why I did that. I wanted to make work of the streamification easier because this way all code in UserStorageManager is already using the correct method (`getUsersStream()`). Providing the `getUsersStream` method is, I would say, out of the scope of this issue. We should have all the streamification work under one issue, therefore I provided a default method for the Stream variant, expecting switching those will be part of the streamification work.